### PR TITLE
misc improvements

### PR DESCRIPTION
**Pull request by arvindkrishnakumar-okta**

### CODE REVIEW

File: direct-auth-samples/src/main/java/com/okta/spring/example/controllers/HomeController.java [hunk 1]
```diff
@@ -104,19 +97,29 @@ public ModelAndView displayIndexOrHomePage(final @RequestParam(name = "interacti
...
         }
 
+        // if error is received in callback, show error page
         if (Strings.hasText(error)) {
             ModelAndView mav = new ModelAndView("error");
             mav.addObject("errors", errorDescription);
             return mav;
         }
 
+        // none of the above cases hold good, so return to root view
```
Comment (JayNewstrom) [inline direct-auth-samples/src/main/java/com/okta/spring/example/controllers/HomeController.java:122]: This comment is a little confusing.

File: direct-auth-samples/src/main/resources/templates/register-password.html [hunk 2]
```diff
@@ -51,13 +51,15 @@
...
 
-<a href="/" class="btn btn-info pull-left">Index Page</a>
+<div style="padding:1em">
+    <a href="/"><i class="fa fa-home fa_custom fa"></i> Home</a>
+</div>
 
 <div class="changepassword-form">
     <form th:action="@{/register-password}" method="post">
-        <h2 class="text-center">Change Password</h2>
+        <h2 class="text-center"><span th:text="${title}"></span></h2>
```
Comment (JayNewstrom) [inline direct-auth-samples/src/main/resources/templates/register-password.html:60]: Should we update the page title too (on line 21)

File: direct-auth-samples/src/main/resources/templates/select-authenticator.html [hunk 3]
```diff
@@ -47,14 +47,16 @@
...
-<a href="/" class="btn btn-info pull-left">Index Page</a>
+<div style="padding:1em">
+    <a href="/"><i class="fa fa-home fa_custom fa"></i> Home</a>
+</div>
 
 <div class="select-authenticators-form">
 
     <form th:action="@{/select-authenticator}" method="post">
-        <h2 class="text-center">Select authenticator</h2>
+        <h2 class="text-center"><span th:text="${title}"></span></h2>
```
Comment (JayNewstrom) [inline direct-auth-samples/src/main/resources/templates/select-authenticator.html:57]: Same as other (consider updating page title on line 21)

**Review by JayNewstrom — APPROVED** [review]

File: direct-auth-samples/src/main/resources/templates/select-authenticator.html (hunk 3 above)
Comment (arvindkrishnakumar-okta) [reply direct-auth-samples/src/main/resources/templates/select-authenticator.html:57]: done

File: direct-auth-samples/src/main/resources/templates/register-password.html (hunk 2 above)
Comment (arvindkrishnakumar-okta) [reply direct-auth-samples/src/main/resources/templates/register-password.html:59]: done

File: direct-auth-samples/src/main/java/com/okta/spring/example/controllers/HomeController.java (hunk 1 above)
Comment (arvindkrishnakumar-okta) [reply direct-auth-samples/src/main/java/com/okta/spring/example/controllers/HomeController.java]: updated :)

**Pull request merged**